README.md

```diff
@@ -91,7 +91,7 @@ The `oLabelsBase` mixin is used to output default label styles, including the `o
 }
 ```
 
-The `oLabelsSize` mixin can be used to output a class for one of the label sizes. Valid sizes are `big` and `small`:
+The `oLabelsSize` mixin can be used to output a class for one of the label sizes. Valid sizes are `big`, `small`, and `wide` (masterbrand only):
```

Comment:
Why is wide reserved for master? (we don't call it `masterbrand` anymore, it's just `master` brand 😬)

Reply:
because currently internal doesn't need it. I know it's only a few extra lines of CSS, but easier to add later than remove.